docs/examples/1d_hist/1d_comparison_only_efficiency.py

```diff
@@ -0,0 +1,37 @@
+"""
+Only the comparison plot
+========================
+
+Example that shows how to only plot the comparison. Here, we compare the ratio between two histograms h1 and h2 when the entries of h1 are a subset of the entries of h2.
+"""
+
+# Set the seed for the .sample() method
+import numpy as np
+
+np.random.seed(8311311)
+
+from plothist import get_dummy_data
+
+df = get_dummy_data()
+
+name = "variable_1"
+
+x_total = df[name][df["category"] == 2]
+x_sample = x_total.sample(frac=0.75) # x_sample is a subset of x_total
```

Comment:
Actually, this code will crash (!) if merged with the main: `df` is not a `pandas` DataFrame anymore, so you cannot use the `sample` method. I would recommend to just take the 75% of the array with a simple array slicing.